assets/js/modules/adsense/components/dashboard/AdBlockingRecoveryWidget.test.js

```diff
@@ -41,9 +41,14 @@ import {
 	SITE_STATUS_ADDED,
 	SITE_STATUS_READY,
 } from '../../util';
+import { getWidgetComponentProps } from '../../../../googlesitekit/widgets/util';
+import { WEEK_IN_SECONDS, DAY_IN_SECONDS } from '../../../../util';
 
 describe( 'AdSenseConnectCTA', () => {
 	let registry;
+	const timestampThreeWeeksAgo = Date.now() - WEEK_IN_SECONDS * 3 * 1000;
```

Comment:
Apologies, I did suggest this but realise the resulting value is in milliseconds rather than the desired seconds. It should be as follows (I'll apply the change to save another round of CR):

```suggestion
	const timestampThreeWeeksAgo = Date.now() / 1000 - WEEK_IN_SECONDS * 3;
```